lib/mv_opentelemetry/ecto.ex

```diff
@@ -56,6 +56,17 @@ defmodule MvOpentelemetry.Ecto do
         _ -> [error: true]
       end
 
+    stacktrace =
+      case meta[:stacktrace] do
+        stacktrace when is_list(stacktrace) ->
+          [
+            stacktrace: Enum.map(stacktrace, &inspect(&1))
```

Comment:
I got 2 questions @maciej-szlosarczyk 

1. Is the attribute name ok or should it be something else?
2. I checked the spec - attribute can be a list, so I turned stacktrace into list of strings. Is that correct or should it be something else as well (e.g. just one string)?

Reply:
1. When it is an error, you'd use "error.stacktrace" (note the namespace). There's also this: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attributes.md#source-code-attributes which might be nicer. Since I ran into this in the past, I'd namespace it to "ecto.stacktrace", so that if there's some other stacktrace it won't conflict with each other.
2. I _think_ it won't work - the support for lists/arrays in attributes was pretty poor, I had to manually convert lists into JSON arrays for Honeycomb. Can you instead make it a long string of strings?